src/client/dfs/obj.c

```diff
@@ -786,6 +790,8 @@ dfs_stat(dfs_t *dfs, dfs_obj_t *parent, const char *name, struct stat *stbuf)
 		oh = parent->oh;
 	}
 
+	DFS_OP_STAT_INCR(dfs, DOS_STAT);
```

Comment:
it sounds like we are incrementing the stat counter twice? once here and once in entry_stat

Reply:
Hmm, yep. Removed this one. The increment only happens in entry_stat() if it succeeded.